engine/runtime/src/test/java/org/enso/compiler/ParseStdLibTest.java

```diff
@@ -215,6 +215,7 @@ public void runBare() throws Throwable {
             "Data/Time/Date_Period.enso",
             "Data/Time/Date_Time.enso",
             "Data/Time/Duration.enso",
+            "Data/Time/Period.enso",
```

Comment:
Sorry for this. One day we shall be able to parse all valid Enso code.

Reply:
Sure, no problem